lib/browsenator/remote/browserstack/window_size.rb

```diff
@@ -3,11 +3,11 @@ module Remote
     module Browserstack
```

Comment:
Now with the new ScreenSize module you can delete this module and all the requires/includes in the remote browsers